doc/host-setup.md

```diff
@@ -28,6 +28,49 @@ You can set the image by passing `--set operator.satelliteSet.kernelModuleInject
 [InitContainer]: https://kubernetes.io/docs/concepts/workloads/pods/init-containers/
 [#137]: https://github.com/piraeusdatastore/piraeus-operator/issues/137
 
+### Build your own injector image
```

Comment:
From this title and the introductory paragraph I had the impression that this would be an image containing pre-built DRBD packages. These instructions are actually for building an injector image that fetches the kernel headers automatically. What should name it? `Build an injector image for compiling without headers on host` is a bit long. `Fedora CoreOS: Build an injector image` since that is its primary purpose?